### PR TITLE
feat: add modfied col and timezone info to schedule col

### PR DESCRIPTION
**Pull request by pkdotson**
### SUMMARY
This pr add modfied to alerts and reports and add timezone information to schedule column

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

after

Screen Shot 2022-05-23 at 3 18 15 PM

### TESTING INSTRUCTIONS
Go to alerts and reports there should be modified column and time zone information in the schedule column.

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Review by hughhhh — APPROVED** [review]

File: superset-frontend/cypress-base/cypress/integration/alerts_and_reports/alerts.test.ts [hunk 1]
```diff
@@ -32,14 +32,17 @@ describe('alert list view', () => {
 
     cy.get('[data-test="listview-table"]').should('be.visible');
     // check alert list view header
+    cy.wait(500);
+    /// cy.get('[data-test="sort-header"]').debug()
```
Comment (lyndsiWilliams) [inline superset-frontend/cypress-base/cypress/integration/alerts_and_reports/alerts.test.ts:36]: There's a debug line here, this should be deleted right?

**Pull request merged**